src/vsr.zig

```diff
@@ -132,7 +132,8 @@ pub const Operation = enum(u8) {
     /// The value 2 is reserved to register a client session with the cluster.
     register = 2,
 
-    /// Operations exported by the state machine (all other values are free):
+    /// Operations <vsr_operations_reserved are reserved for the control plane.
+    /// Operations ≥vsr_operations_reserved are available for the state machine.
     _,
```

Comment:
I _think_ some validation is missing here. If the client sends us operation=92, what will happen? Seems like it'd just blow-up in intToEnum casts. 

I _think_ we should check in `invalid_request` and such that, if operation is < 128, than it's one of the reserved ones. 

:thinking: I think we are not validating this on main either?